collada_parser/include/collada_parser/collada_parser.h

```diff
@@ -41,13 +41,13 @@
 #include <map>
 #include <boost/function.hpp>
 
-#include <urdf_model/model.h>
+#include <urdf_world/types.h>
```

Comment:
This include doesn't exist in Kinect on Wily and Jessie. You should use the compatibility header of the `urdf `package instead to safely define `ModelInterfaceSharedPtr`.